capstone/src/main/java/com/google/step/data/Restaurant.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import java.lang.IllegalArgumentException;
+import java.util.ArrayList;
+import java.util.List;
+
+/** A restaurant to be displayed. */
+public final class Restaurant {
+  private final long restaurantKey;
```

Comment:
Remember how we were talking about building a DAO and having it return Restaurant objects? That means other java code will need to be able to use all the fields in a Restaurant, which in turn means that there will need to be some public way of accessing these fields (e.g. getters).

Reply:
Got it, I added getters, and looking forward to working on the DAO.